sdk/bcs/tests/serde.test.ts

```diff
@@ -2,25 +2,27 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import { describe, it, expect } from 'vitest';
-import { bcs } from './../src/index';
+import { BCS } from './../src/index';
 
 describe('serde', () => {
     it('should serialize primitives in both directions', () => {
+        let bcs = new BCS({});
         let u8 = 100;
         let u64 = '18446744073709551615';
         let u128 = '1844674407370955161518446744073709551';
```

Comment:
I know it just landed, but you might want to include u16, u32, u256 also if you can